lib/Remotery.c

```diff
@@ -769,12 +769,25 @@ static rmtU32 GaloisLFSRNext(rmtU32 value, rmtU32 xor_mask)
         obj = NULL;             \
     }
 
-// New will allocate enough space for the object and call the constructor
-// If allocation fails the constructor won't be called
-// If the constructor fails, the destructor is called and memory is released
-// NOTE: Use of sizeof() requires that the type be defined at the point of call
-// This is a disadvantage over requiring only a custom Create function
-#define rmtTryNew(type, obj, ...)                              \
+
+#define _VA_ARGS_SELECT(PREFIX,_5,_4,_3,_2,_1,SUFFIX,...) PREFIX ## _ ## SUFFIX
```

Comment:
Inspired by info from this thread
https://stackoverflow.com/questions/5588855/standard-alternative-to-gccs-va-args-trick